Gruntfile.js

```diff
@@ -132,66 +122,31 @@ module.exports = function (grunt) {
     'grunt-sass',
     'grunt-nodemon',
     'grunt-concurrent',
-    'grunt-shell',
     'grunt-htmlentities'
   ].forEach(function (task) {
     grunt.loadNpmTasks(task)
   })
 
-  grunt.registerTask('default', [
-    'clean',
-    'copy',
-    'encode_snippets',
-    'sass',
-    'concurrent:target'
-  ])
-
-  grunt.registerTask(
-    'test_default',
-    'Test that the default task runs the app',
-    [
-      'copy',
-      'sass'
-    ]
-  )
+  grunt.registerTask('default', ['clean', 'copy', 'encode_snippets', 'sass', 'concurrent:target'])
 
-  grunt.registerTask(
-    'encode_snippets',
-    'Encode HTML snippets',
-    function () {
-      grunt.task.run('htmlentities')
-    }
-  )
-
-  grunt.registerTask(
-    'lint',
-    'Use govuk-scss-lint to lint the sass files',
-    function () {
-      grunt.task.run('shell', 'lint_message')
-    }
-  )
+  // Encode HTML snippets
+  grunt.registerTask('encode_snippets', ['htmlentities', 'htmlentities_success'])
```

Comment:
Minor: You don't expect to see underscores in Javascript that much so it feels odd to see `encode_snippets` instead of `encode-snippet` but it's not a big deal